src/client.ts

```diff
@@ -624,18 +625,18 @@ export class StreamChat<StreamChatGenerics extends ExtendableGenerics = DefaultG
   }
 
   /**
-	 * testPushSettings - Tests the push settings for a user with a random chat message and the configured push templates
```

Comment:
not sure why linter didn't catch this before @ferhatelmas

Reply:
yes interesting